src/index.html

```diff
@@ -114,6 +114,9 @@ <h2>&rarr; Sponsor Us</h2>
       <h2>&rarr; Slack</h2>
       <p>We discuss Summer of Hacks on the <a href="slack.digitaloxford.com">digital oxford slack</a>. Come and say hi on the #soh channel.</p>
 
+      <h2>&rarr; Twitter</h2>
+      <p>Some short and sweet tweets will be served from <a href="https://twitter.com/Ox_SOH">out twitter account</a>. Let us know anything that's on your mind.</p>
```

Comment:
`Typo`

`out` ➡️ `our`

```suggestion
      <p>Some short and sweet tweets will be served from
      <a href="https://twitter.com/Ox_SOH">our twitter account</a>. Let us know anything that's on your mind.</p>
```

Reply:
:+1: